Makefile

```diff
@@ -1,3 +1,15 @@
 .PHONY: run
 run:
-	python3 pokedex/manage.py runserver
\ No newline at end of file
+	python3 pokedex/manage.py runserver
+

```

Comment:
stubs for future builds which we can use to just populate the pokedex with data from the PokeAPI, to be fully implemented later this week